packages/block-library/src/navigation/edit/index.js

```diff
@@ -283,7 +286,31 @@ function Navigation( {
 		hasResolvedCanUserCreateNavigationMenu,
 	} = useNavigationMenu( ref );
 
+	// Setup initial block insertion state.
```

Comment:
```suggestion
	// The block attempts to retrieve and prioritize any existing navigation menu unless
	// a specific ref is allocated or the user is explicitly creating a new menu. The aim is
	// for the block to "just work" from a user perspective using existing data.
```

Reply:
And I think we can remove the rest of the comment

Reply:
Updated.